components/ledger/pkg/api/controllers/query.go

```diff
@@ -63,3 +64,19 @@ func getBalanceOperator(w http.ResponseWriter, r *http.Request) (storage.Balance
 
 	return balanceOperator, nil
 }
+
+func getCommandParameters(r *http.Request) command.Parameters {
+	previewAsString := r.URL.Query().Get("dryRun")
```

Comment:
why not renaming preview to dry run for variables ? It will be better to understand